src/androidProj/nativeeditplugin/src/main/java/com/bkmin/android/EditBox.java

```diff
@@ -380,7 +380,22 @@ private void Remove()
     private void SetText(String newText)
     {
         if (edit != null) {
+            int cursorPos = edit.getSelectionStart();
+            int previousLength = edit.getText().length();
+
             edit.setText(newText);
+
+            //Update text selection (cursor position) to be the same after editing text, if the user had multiple characters selected, we are losing them and get the cursor in only one position
```

Comment:
Could we shorten the lines for the comments an start them with a space? Would be easier to read them on Github for example.